lib/versions-complete.xml

```diff
@@ -5466,12 +5466,12 @@
 			</library>
 			<library>
 				<file-name>com.liferay.portal.template.velocity.jar!org.apache.velocity.jar</file-name>
-				<version>1.6.4.LIFERAY-PATCHED-1</version>
+				<version>1.6.4-LIFERAY-PATCHED-2</version>
 				<project-name>org.apache.velocity</project-name>
 				<licenses>
 					<license>
-						<license-name>LGPL 2.1</license-name>
-						<license-url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt</license-url>
+						<license-name>Apache License, Version 2.0</license-name>
```

Comment:
@mbowerman Is this change also built by `ant build-lib-versions`?

Reply:
Yes that change was generated by running the command `ant build-lib-versions`. I'm not sure why it made that change.